### PR TITLE
Add route for approving creators.

### PR DESCRIPTION
**Pull request by CalderWhite**
## Notion ticket link

[Approve Creators Route](https://www.notion.so/uwblueprintexecs/Approve-Creators-Route-cbd6c5cc12a94da3ae64e644a47e80ca)

## Implementation description
* Check if admin
* If admin approve by id
* used RESTful URL (`/approve/:id`)

## Steps to test
1. Try to approve a creator without admin
2. Try to approve a creator with admin
3. Check database if the user is an approved creator

## What should reviewers focus on?
* Whether or not these changes propagate to the database

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have run docker-compose up and my project compiled
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: backend/typescript/services/implementations/creatorService.ts [hunk 1]
```diff
@@ -58,6 +58,19 @@ class CreatorService implements ICreatorService {
       throw error;
     }
   }
+
+  async approveCreator(userId: string): Promise<void> {
+    try {
+      const creator = await this.getCreatorById(userId);
+      creator.isApproved = true;
```
Comment (CalderWhite) [inline backend/typescript/services/implementations/creatorService.ts:65]: This is hackey and I'm not sure if the change will propagate to the database. If there is a better way to do this (setters & getters for example) then I would opt to use that method instead.

Comment (Shrinjay) [reply backend/typescript/services/implementations/creatorService.ts:65]: Yeah I'm pretty sure this isn't how our ORM works. Looking at the docs for Sequelize, using a model instance is probably your best bet, use the findByPk method to get the instance from ID then update + save. 

Test it with the following steps:
```
sudo docker exec -it ccbc-db-1 bash
psql -U postgres -d ccbc
select * from users;
```
make sure the users table reflects your changes

**Review by Shrinjay — CHANGES_REQUESTED** [review]

File: backend/typescript/services/implementations/creatorService.ts (hunk 1 above)
Comment (Shrinjay) [reply backend/typescript/services/implementations/creatorService.ts:65]: https://sequelize.org/docs/v6/core-concepts/model-instances/

take a look at userService.ts implementation around line 229

**Comment by CalderWhite** [comment]
Updated to use the orm 👍

**Pull request merged**